media/media.css

```diff
@@ -59,13 +59,59 @@ body {
 
 .container-infos {
   display: flex;
-  flex-wrap: wrap;
   height: calc(100vh - 15rem);
   overflow: scroll;
 }
 
+.templates {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  overflow: scroll;
+}
+
+.templates div {
+  display: flex;
+  flex-wrap: wrap;
+  flex: 1;
+}
+
+.card.template {
+  max-width: 13.75rem;
+  min-width: 10rem;
+  margin: 0 1rem 1rem 0;
+  text-align: center;
+  background-image: linear-gradient(109.6deg, rgba(0,0,0,0.93) 11.2%, rgba(63,61,61,1) 78.9% );
+  align-items: center;
+}
+
+.template .cardTitle, .template .cardTitle {
```

Comment:
I don't like conflicting notations. We have here dash-case (`.container-infos`) and camelCase (`.cardTitle`) in the same file. Can we choose one?